bower.json

```diff
@@ -36,6 +36,11 @@
 		"jquery-1.11.1": "jquery#1.11.1",
 		"jquery-1.11.2": "jquery#1.11.2",
 		"jquery-1.11.3": "jquery#1.11.3",
+		"jquery-1.12.0": "jquery#1.12.0",
+		"jquery-1.12.1": "jquery#1.12.1",
+		"jquery-1.12.2": "jquery#1.12.2",
+		"jquery-1.12.3": "jquery#1.12.3",
+		"jquery-1.12.4": "jquery#1.12.4",
 		"jquery-1.x": "jquery#1.11.3",
```

Comment:
Thanks for pointing this out in your comment (and sorry for taking so long to understand what you were asking). This should be updated to point to 1.12.4; it always tracks the latest 1.x version, and then it gets used in demos and some other places.